colab-notebooks-examples/integrations/openai/intro.ipynb

```diff
@@ -0,0 +1,431 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "690975de",
+   "metadata": {},
+   "source": [
+    "# Create Elastic Cloud deployment\n",
+    "If you don't have an Elastic Cloud deployment, sign up [here](https://cloud.elastic.co/registration?fromURI=%2Fhome) for a free trial.\n",
+    "\n",
+    "* Go to the [Create deployment](https://cloud.elastic.co/deployments/create) page\n",
+    "  * Select Create deployment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de32a789",
+   "metadata": {},
+   "source": [
+    "# Connect to Elasticsearch cloud deployment with python client\n",
+    "\n",
+    "To get started with elasticsearch, we will need to connect to Elastic deployment using the [python client](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/index.html). Since we are using an Elastic Cloud deployment, we will use the Cloud ID to identify our deployment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cb38a868",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python3 -m pip install -q elasticsearch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f7eb391",
+   "metadata": {},
+   "source": [
+    "Instantiate [Elasticsearch python client](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/index.html) by providing cloud id and password of your deployment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3a57b6a8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from getpass import getpass\n",
+    "from elasticsearch import Elasticsearch\n",
+    "\n",
+    "CLOUD_ID = getpass(\"Elastic deployment Cloud ID\")\n",
+    "CLOUD_PASSWORD = getpass(\"Elastic deployment Password\")\n",
+    "client = Elasticsearch(\n",
+    "  cloud_id = CLOUD_ID,\n",
+    "  basic_auth=(\"elastic\", CLOUD_PASSWORD)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a9d6805",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(client.info())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aa9576ca",
+   "metadata": {},
+   "source": [
+    "# Install packages and import modules "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c48bfc91",
+   "metadata": {},
+   "source": [
+    "In this example, we are using wget to download wikipedia vector database and pandas library to read data into a dataframe. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c304b93",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python3 -m pip install openai --quiet\n",
+    "!python3 -m pip install pandas --quiet\n",
+    "!python3 -m pip install wget --quiet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "80b55952",
+   "metadata": {},
+   "source": [
+    "Let's download OpenAI wikipedia vector database and extract to dataframe"
```

Comment:
The downloading and extracting step could be under its own Markdown heading?

Reply:
Yes, will do 👍